awscli/customizations/sso/utils.py

```diff
@@ -56,24 +69,38 @@ def _sso_json_dumps(obj):
 
 def do_sso_login(session, sso_region, start_url, token_cache=None,
                  on_pending_authorization=None, force_refresh=False,
-                 registration_scopes=None, session_name=None):
+                 registration_scopes=None, session_name=None,
+                 fallback_to_device_flow=False):
```

Comment:
Done via https://github.com/aws/aws-cli/pull/8947/commits/e306e3ec67d3f8e3fb1d419f8d148f13f53f16a4 (didn't accept suggestion on GitHub since we had to rename in multiple places).